config/cors.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Cross-Origin Resource Sharing (CORS) Configuration
+    |--------------------------------------------------------------------------
+    |
+    | Here you may configure your settings for cross-origin resource sharing
+    | or "CORS". This determines what cross-origin operations may execute
+    | in web browsers. You are free to adjust these settings as needed.
+    |
+    | To learn more: https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
+    |
+    */
+
+    'paths' => ['api/*', '*', 'sanctum/csrf-cookie'],
```

Comment:
This doesn't makes sense. You're saying "all api paths", "all paths" & "the sanctum csrf cookie path".

Reply:
Isn't the issue that you aren't adding your subdomains to the `allowed_origins` list?